Week3/HW3/3_moreJavascript/6_vehicleAge.js

```diff
@@ -0,0 +1,15 @@
+function vehicleType(color,code, age) {
+    if (code ===1) {
+        type = "car"
+    } else if 
+    (code === 2) {
+        type="motorbike"
+    } else {
+        type = "invalid value"
+    }; 
+    ( age > 1) ? condition = "used" : condition = "new" ;
+    console.log("a" + " " + color +" " + condition + " "+ type)
```

Comment:
Same comments as the exercise `4_vehicleType.js` but in addition, using conditional operators as you did in line 10 isn't the best way.

This is because, conditional operators are best used if we want to assign a variable with a conditional value. Have a read again here about it please - https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/conditional_execution.md

So, a better way of writing this would have been:

```
const condition = (age > 1) ? 'used' : 'new';
```

Reply:
I used 'condition' as a variable actually, not as a conditional operator. But it was not a good one though. 
So, I implemented. 🙂